vowpalwabbit/distributionally_robust.h

```diff
@@ -29,6 +30,30 @@ namespace distributionally_robust
     {
     }
     double qfunc(double w, double r) { return unbounded ? 1 : -(gamma + (beta + r) * w) / ((n + 1) * kappa); }
+
+    void save_load(io_buf& model_file, bool read, bool text)
+    {
+      if (model_file.num_files() == 0) { return; }
+      if (!read || true)
```

Comment:
This will always evaluate to true - why have an if statement?

Reply:
placeholder for model version but forgot to completely remove, good catch